lib/src/block_syntaxes/table_syntax.dart

```diff
@@ -71,29 +72,39 @@ class TableSyntax extends BlockSyntax {
   }
 
   List<String?> _parseAlignments(String line) {
-    final startIndex = _walkPastOpeningPipe(line);
+    final columns = <String?>[];
+    var started = false;
```

Comment:
Could you add a comment above this variable explaining when it is true and when false.

Reply:
Fixed.